src/Layout/PresentationMenu/PresentationSettings/ConnectedPresentationSettings.tsx

```diff
@@ -8,25 +8,35 @@ import {
     changeDisproportionalityIndex,
     toggleShowComparison,
     toggleShowFilters,
+    toggleMergeDistricts,
 } from "../presentation-menu-actions";
 import { DisproportionalityIndex } from "../../Presentation/presentation-models";
+import { ComputationPayload, updateComputation } from "../../../computation";
 
 /**
  * Describes which properties we want to pick from the properties of the
  * component for mapping from state to props.
  */
 interface StateProps
     extends Pick<
-            PresentationSettingsProps,
-            | "currentPresentation"
-            | "districtSelected"
-            | "decimals"
-            | "showPartiesWithoutSeats"
-            | "results"
-            | "disproportionalityIndex"
-            | "showComparison"
-            | "showFilters"
-        > {}
+        PresentationSettingsProps,
+        | "currentPresentation"
+        | "districtSelected"
+        | "decimals"
+        | "showPartiesWithoutSeats"
+        | "results"
+        | "disproportionalityIndex"
+        | "showComparison"
+        | "showFilters"
+        | "year"
+        | "mergeDistricts"
+        | "electionType"
+        | "votes"
+        | "metrics"
+        | "parameters"
+        | "computationPayload"
+        | "settingsPayload"
```

Comment:
Did we actually need to add all of these? I would have thought things like the year would be found from existing properties/objects there. Adding the whole computationPayload and settingsPayload seems a bit excessive as well, but perhaps I've misunderstood what's going on?